easier68k/core/opcodes/bcc.py

```diff
@@ -0,0 +1,737 @@
+    Additionally, a redefinition of command_matches(), from_str(), and disassemble_instruction() will suffice
+
+
+    BCC:     Operation: If Condition True Then PC + dn → PC
+
+    Syntax:  Bcc < label >
+
+    Size = (Byte, Word, Long*)
+    *(MC68020, MC68030, and MC68040 only)
+
+    Description: If the specified condition is true, program execution continues at location (PC) + displacement. The program counter contains the address of the instruction word for
+    the Bcc instruction plus two. The displacement is a twos-complement integer that
+    represents the relative distance in bytes from the current program counter to the
+    destination program counter. If the 8-bit displacement field in the instruction word is
+    zero, a 16-bit displacement (the word immediately following the instruction) is used. If
+    the 8-bit displacement field in the instruction word is all ones ($FF), the 32-bit
+    displacement (long word immediately following the instruction) is used. Condition code
+    cc specifies one of the following conditional tests (refer to Table 3-19 for more
+                                                         information on these conditional tests):
+
+    Condition Codes:    Not affected.
+
+    Instruction Fields:
+        Condition field—The binary code for one of the conditions listed in the table.
+        8-Bit Displacement field—Twos complement integer specifying the number of bytes
+            between the branch instruction and the next instruction to be executed if the
+            condition is met.
+        16-Bit Displacement field—Used for the displacement when the 8-bit displacement
+            field contains $00.
+        32-Bit Displacement field—Used for the displacement when the 8-bit displacement
+            field contains $FF.
+
+    NOTE    A branch to the immediately following instruction automatically
+    uses the 16-bit displacement format because the 8-bit
+    displacement field contains $00 (zero offset).
+    """
+
+    operand = None
+    address = None
+    cond_code = None
+    offset = None
+    size = None
+    
+    def __init__(self, params: list):
+        assert len(params) == 2
+        assert isinstance(params[0], int)
+        assert isinstance(params[1], int)
+
+        # Operand = target address
+        self.operand = params[0]
+        self.address = params[1]
+
+        # make offset out of operand
+        self.offset = int(self.operand) - int(self.address + 2)
+
+        # get the size
+        self.size = OpSize.BYTE if int(self.offset) < 128 else \
+            (OpSize.WORD if int(self.offset) < 326778 else OpSize.LONG)
+
+        # offset to bits
+        self.offset = Bits(int=self.offset,
+                           length=(self.size.get_number_of_bytes() * 8)).int
+
+        # Note on Page 130 of the NXP 68K Manual
+        # 0x00 8 bit offset defaults to word size
+        if self.offset == 0:
+            self.size = OpSize.WORD
+
+;    @astractmethod
```

Comment:
```suggestion
    @astractmethod
```
Found your build error

Reply:
if this depends on #132 , then let's just remove it from this PR and I'll do it in there